config/environments/development.rb

```diff
@@ -36,6 +36,9 @@
   # Raises helpful error messages.
   config.assets.raise_runtime_errors = true
 
+  # Allow debugging over docker networks
+  config.web_console.whitelisted_ips = '172.21.0.1'
```

Comment:
What is this for? Why do we need to whitelist a private ip?

Reply:
Aahhh right this shenanigans. I think this has something to do with how docker handles networking internally, and solved some weird web console problem. I didn't really take the time to figure out what was actually going on at the time, and intended to pull the line and put in an issue once/if this pr got merged. Then I promptly forgot, and added it to whichever commit that was.

It looks like this is from a private ip address block [(172.16.0.0/12)](https://en.wikipedia.org/wiki/Private_network), so I don't feel _too_ bad about whitelisting it.

That said, this was added in attempt to do some debugging, and did get rid of an error message. Unfortunately, I think that this value potentially varies from machine to machine within the block, and therefore I think the appropriate solution is either:

1. Whitelisting the entire ip block
1. Disabling it

Again, I'm not totally sure what's going on here (or if we can wildcard the entire block), so resolving this properly will involve:

1. Figuring out what this is actually allowing us to debug
1. If we want access, how can we whitelist an entire block, and what is that block?
1. What are the potential security ramifications of whitelisting a private ip block?
1. Do we care about said security ramifications for a dev build?

As for what to do right now, I'm tempted to leave it in because it seems fairly minor, this isn't deployed anywhere, and I don't want to add another commit to this branch. That said removing it is totally doable.